nf_core/pipeline-template/conf/test.config

```diff
@@ -16,8 +16,8 @@ params {
 
     // Limit resources so that this can run on GitHub Actions
     max_cpus   = 2
-    max_memory = 6.GB
-    max_time   = 6.h
+    max_memory = '6.GB'
+    max_time   = '6.h'
```

Comment:
So are we doing that or not?

Reply:
Yes, we are. I found a really weird bug when submitting pipelines on Tower where the settings in `test.config` were not being propagated to the run. Need to investigate more but this is fixed it and is the safest option if we want to get a release out yesterday.

Reply:
See comment on the dev PR but this was also for JSON schema validation. This way Nextflow doesn't switch the units around so it fails less.